### PR TITLE
remove axioms in copsex2g; revise opelopabsbALT

### PR DESCRIPTION
**Pull request by icecream17**
see comment of commit

This changes brabsb2 from Rodolfo Medina's mathbox

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -50593,9 +50593,20 @@ That theorem bundles the theorems ( ` |- E. x ( x = y -> z e. x ) ` with
     $d x y ps $.  $d x y A $.  $d x y B $.
     copsex2g.1 $e |- ( ( x = A /\ y = B ) -> ( ph <-> ps ) ) $.
     $( Implicit substitution inference for ordered pairs.  (Contributed by NM,
-       28-May-1995.) $)
+       28-May-1995.)  Use a similar proof to ~ copsex4g to reduce axioms.
+       (Revised by SN, 1-Sep-2024.) $)
```
Comment (benjub) [inline set.mm:50597]:
```suggestion
       28-May-1995.)  Use a similar proof to ~ copsex4g to reduce axiom
       usage.  (Revised by SN, 1-Sep-2024.) $)
```

**Review by benjub — CHANGES_REQUESTED** [review]

**Comment by benjub** [comment]
Does this complete what you wrote in https://github.com/metamath/set.mm/pull/4176#issue-2497834645, e.g., for brabga ?

If you don't have time or motivation to do elrab2gw, maybe you can simply add a comment to elrab2g, e.g.,
> It is possible to prove a weaker version "elrab2gw" not requiring ~ ax-10 , ~ ax-11 , ~ ax-12 , as is ~ elab2gw to ~ elab2g , and use that weaker version instead of ~ elrab2g in most of its applications.

**Review by tirix — APPROVED** [review]

**Comment by benjub** [comment]
> Does this complete what you wrote in [#4176 (comment)](https://github.com/metamath/set.mm/pull/4176#issue-2497834645), e.g., for brabga ?
> 
> If you don't have time or motivation to do elrab2gw, maybe you can simply add a comment to elrab2g, e.g.,
> 
> > It is possible to prove a weaker version "elrab2gw" not requiring ~ ax-10 , ~ ax-11 , ~ ax-12 , as is ~ elab2gw to ~ elab2g , and use that weaker version instead of ~ elrab2g in most of its applications.

@icecream17 I don't know how to interpret your :+1: :
* what about brabga ?
* will you do the same work with elrab2gw ? or will you add a kind of comment to it as I suggested ?

**Comment by icecream17** [comment]
+ ~ brabga indeed does not depend on ax-10 -- ax-12 anymore. (This is what I meant by 👍; my mind was kind of in a trance so I didn't notice the ambiguity)
+ I ended up adding elrab2w (label ~ elrab2 does not have g), and used it in one theorem: https://github.com/metamath/set.mm/pull/4185 (relevant commit is the last one). It doesn't seem like there is as much use for elrab2w though... perhaps just having a comment might be a better idea?

(Screenshot with Ctrl+F for theorems using elrab2 that start with `el`:)
image

---

**Edit:** better explanation

I have [many TODOs](https://gist.github.com/icecream17/c85a1288ccebd09e21c7609943a9a8d6), which can be rather hard to keep track of.

I'm trying to do about one or two at a time, in separate prs. Usually, any particular TODO leads to more things, so this is a good unit of measurement to make sure that any particular pr is understandable and isolated.

**Review by benjub — APPROVED** [review]

**Pull request merged**